application/backend/src/main/resources/apps/bulma/components/page/body.html

```diff
@@ -16,9 +16,11 @@
   ~
 */-->
 
+<sly data-sly-use.model="pl.ds.bulma.components.models.PageComponent">
 <sly data-sly-test.navbarFixed="${properties.navbarFixed}"></sly>
 <sly data-sly-test.position="${properties.navbarFixedPosition}"></sly>
 
-<body class="${properties.defaultTemplateCSS} ${navbarFixed ? position : ''}">
+<body data-config="{'contactForm': '${model.configEndpoint}'}" class="${properties.defaultTemplateCSS} ${navbarFixed ? position : ''}">
```

Comment:
I had to change ' with " to make it work:
`data-config='{"contactForm": "${model.configEndpoint}"}'`
I'm not sure if it's only my issue?